lib/generators/spotlight/templates/config/initializers/spotlight_initializer.rb

```diff
@@ -70,13 +70,16 @@
 # (https://console.cloud.google.com/iam-admin/iam -> Service accounts -> click on service account -> keys)
 # c) set ga_property_id below to your site's property id (located in admin -> Property -> Property details upper right hand corner)
 # d) Set the ga_web_property_id. (located in admin -> Data collection and modification -> Web stream details and begins with G-)
+# e) (optional) set ga_date_range. This allows you throttle the dates the user can filter by.
+# ga_date_range values should use a Date object i.e. (Date.new(YYYY, MM, DD)).
```

Comment:
Is this true? It looks like they should be strings like `YYYY-MM-DD` to me.

Reply:
Everywhere else these dates are strings so in the validation step when you try to compare a date object set in the config with the string "date" from the params I think you'll get an error. It also seems critical that the date format set here is 'YYYY-MM-YY' otherwise the string comparisons in the validate step won't evaluate correctly.